localstack/services/kinesis/kinesis_listener.py

```diff
@@ -304,6 +292,29 @@ def decode_content(self, data, describe=False):
 
         return decoded
 
+    def create_event_record(self, data, sequence_number):
+        # Note: avoid adding 'encryptionType':'NONE' in the event_record, as this breaks .NET Lambdas
+        return {
+            "approximateArrivalTimestamp": epoch_timestamp(),
+            "data": data["Data"],
+            "partitionKey": data["PartitionKey"],
+            "sequenceNumber": sequence_number,
+        }
+
+    def check_for_dynamodb_source_arn(self, event_record):
+        try:
+            decoded = json.loads(base64.b64decode(event_record["data"]))
+            if "dynamodb" in decoded:
+                return decoded.get("eventSourceARN")
+            else:
+                return None
+        except Exception:
+            return None
+
+    def process_kinesis_records(self, event_records, stream_name):
```

Comment:
please rename this function to indicate that it handles kinesis records _and_ dynamodb records